okhttp/src/main/java/com/squareup/okhttp/OkHttpTlsUpgrader.java

```diff
@@ -56,10 +53,7 @@
   // TODO(madongfly): We should only support "h2" at a right time.
   private static final List<String> SUPPORTED_HTTP2_PROTOCOLS = Collections.unmodifiableList(
       Arrays.asList("h2", "h2-14", "h2-15", "h2-16"));
-
-  // A dummy address used to bypass null check.
-  private static final InetSocketAddress DUMMY_INET_SOCKET_ADDRESS =
-      InetSocketAddress.createUnresolved("fake", 73);
+  private static final List<Protocol> TLS_PROTOCOLS = Arrays.<Protocol>asList(Protocol.HTTP_2);
 
```

Comment:
This list is mutable, so it shouldn't be in all caps.

Better would be wrap it with collections.unmodifiablelist